features/networking/ptp_operator.feature

```diff
@@ -11,3 +11,102 @@ Feature: PTP related scenarios
       | app=linuxptp-daemon |
     And status becomes :running of exactly 1 pods labeled:
       | name=ptp-operator |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-25738
+  @admin
+  Scenario: Linuxptp run as slave can sync to master
+    Given the ptp operator is running well
+    Given I switch to the first user
+    Given I store the nodes in the clipboard
+    Given I obtain test data file "networking/ptp/ptp_master.yaml"
+    When I run oc create as admin over "ptp_master.yaml" replacing paths:
+      | ["spec"]["recommend"][0]["match"][0]["nodeName"] | <%= cb.nodes[0].name %> |
+    Then the step should succeed
+    Given I obtain test data file "networking/ptp/ptp_slave.yaml"
+    When I run oc create as admin over "ptp_slave.yaml" replacing paths:
+      | ["spec"]["recommend"][0]["match"][0]["nodeName"] | <%= cb.nodes[1].name %> |
+    Then the step should succeed
+    Given I register clean-up steps:
```

Comment:
Register clean-up steps should happen before those resources are created.

Reply:
@liangxia Sorry, I don't get this. I checked other cases using register clean-up steps,  this step was used after create resource.  Do you mean I need move "Given I register clean-up steps:" to before line 22?

Reply:
Because there are multiple resources created. Just think that if the second creation failed due to any reason, the clean-up step haven't registered successfully yet, then we are leaving unexpected resources in the cluster.

Reply:
Thanks, fixed it.